specification/paths/Brokers-broker_id-relationships-carriers.json

```diff
@@ -11,6 +11,7 @@
     "security": [
       {
         "OAuth2": [
+          "broker.member",
```

Comment:
Why are we adding the broker.member scope to the POST carrier relationship endpoint?

Reply:
Whoops, good catch! This was supposed to be set on the `GET /brokers/{broker}/carriers` endpoint.